packages/wonder-blocks-dropdown/components/dropdown-core-virtualized.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+import * as React from "react";
+import {VariableSizeList as List} from "react-window";
```

Comment:
I was a little surprised by the rename here, I think it's ok to leave the name as-is.